README.md

```diff
@@ -20,6 +20,15 @@ Run the importer tool there:
 $ git-stats-importer
 ```
 
+### Import only selected commits
+
+You can use this bash variant of import to import only commits from specific author,
+time, grep or whatever you can formulate as git log parameters.
+
+```sh
+$ git-stats-importer-custom --help
```

Comment:
To have this available after a global install/update, you have to add the following line [in the `bin` field from `package.json`](https://github.com/IonicaBizau/git-stats-importer/blob/8335a3c69b3b851abb9f79c9ca8cdd62f727a8ff/package.json#L7):

```js
...
"bin": {
   ...,
   "git-stats-importer-custom": "./bin/git-stats-importer-custom"
}
...
```

Also, make sure that the script has executable permissions (if not, do `chmod +x git-stats-importer-custom`).